packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.scss

```diff
@@ -0,0 +1,20 @@
+/* stylelint-disable max-nesting-depth */
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+:local {
+  .container {
```

Comment:
@mjhenkes mentioned our team may want to use this component. If we do, these styles will have to be optional or moved to a wrapper component. Our embedded components are not 100% height.